docs/docusaurus/docs/core/connect_to_data/configure_credentials/_key_pair.md

```diff
@@ -1,3 +1 @@
-If you're connecting to Snowflake, you can use key-pair authentication instead of a password. This improves security and can be helpful for automations.
-
-Follow Snowflake's docs to [configure and store the private and public keys](https://docs.snowflake.com/en/user-guide/key-pair-auth). 
\ No newline at end of file
+To store the value for `private_key`, follow Snowflake's docs to [configure and store private and public keys](https://docs.snowflake.com/en/user-guide/key-pair-auth). Use Environment Variables or `config.yml` to store your other connection details.
```

Comment:
Similar to my comment above, this should be updated to read that env variables and and the config parameter store can be used to store user's private key. I wonder if this part can just be dropped altogether, since private_key will no longer be a special case.